templates/prescription/general_edit.html

```diff
@@ -149,90 +156,92 @@
   </td>
 </tr>
 <tr>
-	<td COLSPAN="1" class="text" ALIGN="right" VALIGN="MIDDLE" >{xl t='Starting Date'}</td>
-	<td COLSPAN="2" ALIGN="LEFT" VALIGN="MIDDLE" >
-		{html_select_date start_year="-10" end_year="+5" time=$prescription->start_date prefix="start_date_"}
-	</td>
+  <td COLSPAN="1" class="text" ALIGN="right" VALIGN="MIDDLE" >{xl t='Starting Date'}</td>
+  <td COLSPAN="2" ALIGN="LEFT" VALIGN="MIDDLE" >
+    {html_select_date start_year="-10" end_year="+5" time=$prescription->start_date prefix="start_date_"}
+    <input type="hidden" name="start_date" value="{$prescription->start_date}">
```

Comment:
Did you add the above line for a bug fix? This is a new line of code. Note it's easier to see these things when look at this code while ignoring whitespace which can do with following link(note the `?w=1` at end of link):
https://github.com/openemr/openemr/pull/1017/files?w=1

Reply:
No, I did not add that line of code.  I have not recently edited this file.

Reply:
git is clearly showing the above line is not in main codebase, but is in your code. If you did not add that line, then rec removing it.
If you want to get a clearer idea of what code you added to this file, check out the diff which ignores whitespace changes:
https://github.com/openemr/openemr/pull/1017/files?w=1

Reply:
Here is my proof that I did not add the lines 159-162. In the image below, I went back to where I did the dragndrop mod which has been untouched since then. The window on the left shows the file date. Look at the top of the editor, you can see that I am in the dragdrop folder. In the editor are the same lines from April of this year. This is from the main code base. I always fetch before creating a new branch. That code has been there long before April and I did not add it.  Can you be clear for me, you want me to remove lines 159-162?
![not_me](https://user-images.githubusercontent.com/834958/30127001-c0d8990e-930b-11e7-9d24-8cd4d77ae2c1.png)

Reply:
only 1 line at 162. don't worry about this; it's minor and i'll fix it up when get this into the codebase (have fingers crossed that it will get into the codebase tonight :) )